[AppBar] Fix background color on dark mode (#26545)
mui-org/material-ui: docs/src/pages/components/app-bar/app-bar.md

  );
}
```

> ## Enable Color on Dark

Reviewer: Our heading capitalization convention:

```suggestion
## Enable color on dark mode
```

Reviewer: We might want to explain what "on dark" mean 

```suggestion
## Enable color on dark mode
```

> From v5 onward, `color` prop has no effect on dark mode according to [material design spec](https://material.io/design/color/dark-theme.html). You can opt out by passing `enableColorOnDark` prop.

Reviewer: Prefer breaking lines between sentences, makes reviewing easier:

```suggestion
From v5 onward, `color` prop has no effect on dark mode according to [material design spec](https://material.io/design/color/dark-theme.html).
You can opt out by passing `enableColorOnDark` prop.
```

Reviewer: We usually don't mention previous versions:

```suggestion
The `color` prop has no effect on dark mode according to [material design spec](https://material.io/design/color/dark-theme.html).
You can opt out by passing `enableColorOnDark` prop.
```

Reviewer: @mbrookes Are you OK to settle on this terminology?

```suggestion
The `color` prop has no effect on dark mode according to [Material Design guidelines](https://material.io/design/color/dark-theme.html).
You can opt out by passing `enableColorOnDark` prop.
```

Reviewer 2: > @mbrookes Are you OK to settle on this terminology?

No, but it seems that it was merged before you asked 🙂.

A gentle reminder to @mui-org/maintainers to tag me on any PRs with new or significantly changed language or terminology.

Reviewer: @mbrookes You have another shoot at it in #26628.
